packages/jellyfish-api-core/__tests__/category/blockchain/waitForNewBlock.test.ts

```diff
@@ -0,0 +1,53 @@
+import { MasterNodeRegTestContainer } from '@defichain/testcontainers'
+import { ContainerAdapterClient } from '../../container_adapter_client'
+
+const container = new MasterNodeRegTestContainer()
+const client = new ContainerAdapterClient(container)
+
+describe('new block height 10', () => {
+  beforeAll(async () => {
+    await container.start()
+  })
+
+  afterAll(async () => {
+    await container.stop()
+  })
+
+  it('should wait for new block height', async () => {
+    {
+      const count = await client.blockchain.getBlockCount()
+      expect(count).toStrictEqual(0)
+    }
+
+    {
+      const promise = client.blockchain.waitForBlockHeight(10)
```

Comment:
hmm.. the test file name is not right...

Reply:
😰